### ISSUE TITLE
(SSBOT 0.6) Error when I authorize or I send a tweet

### ISSUE THREAD

**Issue by Djibs** [zachwhalen/ssbot#14]
I just installed version 0.6 and I have an error when I want to give permission to twitter or when I want to send a test tweets.

See the attached screen printer

**Comment by Djibs** [comment]
=> https://i.imgur.com/VwxIy1p.jpg

**Comment by j2l** [comment]
I habe the same issue.
Did you find a fix?

**Comment by zachwhalen** [comment]
I think I found the problem and [hopefully fixed it with this little patch](https://github.com/zachwhalen/ssbot/commit/36694200434f9dd470c1e21098b0545a8c3f434a). I've only tested it in one scenario, but it seems to work. I'll leave this issue open for now until someone else can confirm that this solves the problem for them.

If it indeed it works, I'll update the linked Version 0.6.0 Document from the main page on this repo.
